tcms/rpc/api/testexecution.py

```diff
@@ -131,7 +131,15 @@ def history(execution_id):
         :raises PermissionDenied: if missing *testruns.view_testexecution* permission
     """
     execution = TestExecution.objects.get(pk=execution_id)
-    execution_history = execution.history.all().order_by('-history_date').values()
+    execution_history = execution.history \
+        .all() \
+        .order_by('-history_date') \
```

Comment:
if you start the new lines after an opening bracket then you don't need to escape them, see the call to `.values()` below.